### ISSUE TITLE
issues with bootstrap.sh

### ISSUE THREAD

**Issue by diwathenettai** [aws-quickstart/quickstart-oracle-database#5]
The bootstrap.sh seems to be failing to make an entry onto the /etc/hosts file.
Also I tried to edit it and not able to - just wondering if there is any checksum kind of operation on this file to make it fail if there are any changes that are made at the time of testing.
Please let me know.

Thank you,
K.Diwakar

**Comment by StylusEater** [comment]
I do not believe the hostname is set by default. I had to add entries to UserData to address this issue. Maybe @kamarja would want a patch to address this issue?

**Comment by StylusEater** [comment]
@diwathenettai did you see the response? Otherwise, maybe this issue should be closed @kamarja ?

**Comment by kamarja** [comment]
@StylusEater Yes I will make the patch and close the issue.
